### PR TITLE
MC-10744 (2): Distinguish deeply nested sections, format cURL

### PR DESCRIPTION
**Pull request by patrickspensieri**
#### Changes made
- indent tables to have a clear separation between deeply nested sections
- use examples instead of placeholders in all cURL requests
- other small changes, such as using the same cURL formats throughout

#### UI changes
*Before (nested sections)*
1-sections

*After (nested sections with margin)*
1 3-margin

#### Related PRs
- #132 
- #134

### CODE REVIEW

**Review by sweetsweetstefan — APPROVED** [review]
LGTM

File: source/includes/gcp/_k8_pods.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ curl -X GET \
 
 Retrieve a list of all pods in a given [environment](#administration-environments)
```
Comment (lamminade) [inline source/includes/gcp/_k8_pods.md:211]: sentences describing what the api call does have periods inconsistently, for example in the cloudstack network acls documentation

**Review by lamminade — APPROVED** [review]
looks great, thanks for all your work

File: source/includes/gcp/_k8_pods.md (hunk 1 above)
Comment (patrickspensieri) [reply source/includes/gcp/_k8_pods.md:211]: Fixed all of them [here](https://github.com/cloudops/cloudmc-api-docs/pull/134/commits/996dc0a649c40dabc549d50add9dc3ba75bebec9)

**Pull request merged**